packages/ui/src/components/sha-badge.tsx

```diff
@@ -44,13 +40,15 @@ function Icon({ ...props }: IconProps) {
   const { children, handleClick } = props
 
   return (
-    <button
-      className="flex items-center border-l bg-background px-1.5 py-0.5 hover:bg-muted/50"
+    <Button
```

Comment:
One for another time, but for a11y reasons, we probably should only use a `button` if there is a click handler, otherwise assistive technologies will announce this as interactive, when it doesn't actually do anything. Perhaps we can double back later and change this to use something like a div instead of a button when `handleClick` isn't passed 🤔

Reply:
Added this behavior.